doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,175 @@
+
+Building for the application core only
+======================================
+
+Building for the application core follows the default building process for the |NCS|.
+For detailed instructions, refer to the :ref:`building` page.
+
+.. _building_nrf54h_app_flpr_core:
+
+Building for both the application and FLPR core
+===============================================
+
+Building for both the application core and the FLPR core differs from the default |NCS| procedure.
+Additional configuration is required to enable the FLPR core.
+
+This section explains how to build and program both cores, covering separate builds and sysbuild configurations.
+The FLPR core supports two variants:
+
+* ``nrf54h20dk/nrf54h20/cpuflpr``: FLPR runs from SRAM (recommended method).
+  The application core image must include the ``nordic-flpr`` :ref:`snippet <app_build_snippets>`.
+
+* ``nrf54h20dk/nrf54h20/cpuflpr/xip``: FLPR runs from MRAM.
+  The application core image must include the ``nordic-flpr-xip`` snippet.
+
+Standard build
+--------------
+
+This subsection explains how to build an application using :ref:`sysbuild <configuration_system_overview_sysbuild>`.
+
+.. note::
+   Currently, the documentation does not provide specific instructions for building an application image using sysbuild to incorporate the FLPR core as a sub-image.
+   The only documented scenario involves building the FLPR as the main image and the application as a sub-image.
+
+Follow these steps to complete the build:
+
+.. tabs::
+
+   .. group-tab:: Using minimal sample for VPR bootstrapping
+
+      This option automatically programs the FLPR core with :ref:`dedicated bootstrapping firmware <vpr_flpr_nrf54h20_initiating>`.
+
+      To build and flash both images, run the following command to perform a :ref:`pristine build <zephyr:west-building>`:
+
+      .. code-block:: console
+
+         west build -p -b nrf54h20dk/nrf54h20/cpuflpr
+         west flash
+
+   .. group-tab:: Using an application that supports multi-image builds
+
+      If your application involves creating custom images for both the application core and the FLPR core, disable the VPR bootstrapping sample by setting the ``SB_CONFIG_VPR_LAUNCHER`` option to ``n`` when building for the FLPR target.
+      For more details, see :ref:`how to configure Kconfig <configuring_kconfig>`.
+
+      To build and flash both images, run the following command to perform a :ref:`pristine build <zephyr:west-building>`:
+
+      .. code-block:: console
+
+         west build -p -b nrf54h20dk/nrf54h20/cpuflpr -- -DSB_CONFIG_VPR_LAUNCHER=n
+         west flash
+
+Separate images
+---------------
+
+You can build and program the application sample and the FLPR sample as separate images using either the |nRFVSC| or the command line.
+To use nRF Util, see `nRF Util`_.
+Depending on the method you select, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
```

Comment:
Not familiar with VSCode flow, needs to be validated by someone else

Reply:
From the point of view of vscode The user can use create a separate build configuration and select flipr as the target.

In that case the command we run will be 

west build --build-dir <build> <app-dir> -pristine -target nrf54h20dk/nrf54h20/cpuflpr .....

One minor comment I why I sysbuild not being used? was sysbuil not created for such a case?